misc/de-metas-common/de-metas-common-externalsystem/src/main/java/de/metas/common/externalsystem/ExternalSystemConstants.java

```diff
@@ -73,4 +73,6 @@
 
 	public static final String HEADER_PINSTANCE_ID = "x-adpinstanceid";
 	public static final String HEADER_EXTERNALSYSTEM_CONFIG_ID = "x-externalsystemconfigid";
+
+	public static final int DEFAULT_ORDER_PAGE_SIZE = 100;
```

Comment:
```suggestion
	public static final int DEFAULT_SW6_ORDER_PAGE_SIZE = 100;
```

it's OK from my pov to tos start out with a hardcoded value. but i think it's doesn't make sense to presume this a valid constant for all sorts of orders